src/elements/core/testing/private/describe-each.ts

```diff
@@ -2,7 +2,7 @@ function generateDescribeName(payload: Record<string, unknown>): string {
   return Object.entries(payload)
     .map(
       ([key, value]) =>
-        `${key}=${typeof value === 'object' && value ? `(${generateDescribeName(value as Record<string, unknown>)})` : value}`,
+        `${key}=${typeof value === 'object' && !(value instanceof Date) && value ? `(${generateDescribeName(value as Record<string, unknown>)})` : value instanceof Date ? `${value.valueOf()}` : value}`,
```

Comment:
```suggestion
        `${key}=${ value instanceof Date ? `${value.valueOf()}` : typeof value === 'object' && value ? `(${generateDescribeName(value as Record<string, unknown>)})` : value}`,
```
A little improvement. 
Although, if in the future we would have to update this code again, we should probably refactor it in a separate method

Reply:
I'm fixing as suggested;
using `value.toISOString()` breaks the build due to the presence of the dot in the timestamp.